src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -559,17 +606,19 @@ private static void parseIncludeVariables(Document doc) throws XPathExpressionEx
 
                 if (includeFileName == null || includeFileName.trim().isEmpty()) {
                     log.warn("Unable to resolve include file location "+nodeValue+". Skipping the included file during application location processing.");
-                    return;
+                    continue;
                 }
-                    
-                Document docIncl = getIncludeDoc(includeFileName);
-
-                if (docIncl != null) {
-                    parseVariablesForBothValues(docIncl);
-                    // handle nested include elements
-                    parseIncludeVariables(docIncl);
-                } else {
-                    log.warn("Unable to parse include file "+includeFileName+". Skipping the included file during application location processing.");
+
+                ArrayList<Document> inclDocs = getIncludeDoc(includeFileName);
+
+                for (Document inclDoc : inclDocs) {
+                    if (inclDoc != null) {
+                        parseVariablesForBothValues(inclDoc);
+                        // handle nested include elements
+                        parseIncludeVariables(inclDoc);
+                    } else {
+                        log.warn("Unable to parse include file "+includeFileName+". Skipping the included file during application location processing.");
```

Comment:
It seems we no longer return `null` for Documents now, and are already properly logged in other methods when initially parsing the Document.
Will remove this redundant/misleading log statement, and possibly the whole if/else statement.

Reply:
The `getIncludeDoc` method used to return null for  the single file. Now I presume it could return an empty list? Maybe check for that and log it. But make sure that proper logging is done when a file within a dir is not able to be found/processed for some reason.